packages/core/forms/src/components/FormGenerator.vue

```diff
@@ -11,8 +11,21 @@
         v-for="field in fields"
         :key="field.model"
       >
+        <form-redis
+          v-if="field.model === '__redis_partial' && enableRedisPartial"
```

Comment:
Why `__redis_partial`? It looks very wired. Is there any particular consideration?